common/src/main/java/com/microsoft/identity/common/internal/util/PackageUtils.java

```diff
@@ -145,6 +148,7 @@ public static final String verifySignatureHash(final @NonNull List<X509Certifica
             hashListStringBuilder.append(signatureHash);
             hashListStringBuilder.append(',');
 
+            //The next() functionality of the iterator should be overridden to retrieve the signatureHash of the entry.
```

Comment:
What does this mean?

Reply:
That's kind of fundamental to the way iterators work.  hasNext() tells you if there is another element to consider, next() fetches it and updates the iterator state. <shrug>

Reply:
oh in this case, the next fetches the signature hash directly and not the element, I can go ahead and remove the comment, but just thought of pointing it out for the next time someone, uses this and passes in an iterator.

Reply:
OK, maybe "This iterator should traverse a list of signature hash elements".  You don't * need * to override next.  I mean, I could write elements.stream().map(e -> e.signatureHash).collect(Collectors.toList()).iterator(), and that shoud be fine, if a bit silly.